agent/taskresource/logrouter/logrouter_unix.go

```diff
@@ -89,6 +89,41 @@ func NewLogRouterResource(cluster, taskARN, taskDefinition, ec2InstanceID, dataD
 	return logRouterResource
 }
 
+// GetCluster returns the cluster
+func (logRouter *LogRouterResource) GetCluster() string {
```

Comment:
where are these getters used? I see them only in tests

Reply:
yeah, they are only used in tests in the api/task package. None of the existing task resources exposes public fields, so i keep these fields private and use getter.

Reply:
ok, could you add a comment mentioning these are used only for tests for now? I don't want us to accidentally use them without locks.

Reply:
sure. added comments for each of these methods